kubernetes/samples/scripts/create-weblogic-domain/manually-create-domain/domain.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# This is an example of how to define a Domain resource.  Please read through the comments which explain
+# what updates are needed.
+#
+apiVersion: "weblogic.oracle/v2"
+kind: Domain
+metadata:
+  # Update this with the `domainUID` of your domain:
+  name: domain1
+  # Update this with the namespace your domain will run in:
+  namespace: weblogic
+  labels:
+    weblogic.resourceVersion: domain-v2
+    # Update this with the `domainUID` of your domain:
+    weblogic.domainUID: domain1
+
+spec:
+  # This parameter provides the location of the WebLogic Domain Home (from the container's point of view).
+  # Note that this might be in the image itself or in a mounted volume or network storage.
+  domainHome: /u01/oracle/user_projects/domains/domain1
```

Comment:
Shall we add a comment saying "Update this with the in-pod location of your WebLogic domain home"?